### PR TITLE
[T5A1][T16-B1] Ong Li-Jen

### PR DESCRIPTION
**Pull request by lijen1994**
Made changes to DeveloperGuide.md to add functions and user stories.

### CODE REVIEW

**Comment by lijen1994** [comment]
Ready for review

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -54,6 +54,31 @@ Priority | As a ... | I want to ... | So that I can...
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+#### Use case: Rename a persons tag
+
+**MSS**
+
+1. User requests to rename a specific person's tag
```
Comment (PierceAndy) [inline doc/DeveloperGuide.md]: As mentioned in the tutorial, the use case should not have been about renaming specific tags tied to a single person (though this is not a huge issue).

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -54,6 +54,31 @@ Priority | As a ... | I want to ... | So that I can...
...
+2. Addressbook shows the persons tag if any.
+3. AddressBook prompts the user to input in the new tag.
+4. User inputs new tag (e.g. 'friends' to buddies)
+5. AddressBook upon completing input will request for confirmation.
+6. Upon user confirmation, the tag in AddressBook will be renamed.
```
Comment (PierceAndy) [inline doc/DeveloperGuide.md]: Every step should clearly show who does what. Avoid using the passive voice. Who does the renaming? Would be better phrased as:
`6. User confirms edit.`
`7. AddressBook renames tag.`

**Comment by PierceAndy** [comment]
@lijen1994 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**